### ISSUE TITLE
Oculus Quest2 different rings in each eyes

### ISSUE THREAD

**Issue by huliqan** [fholger/openvr_foveated#6]
**Which game is affected?**
IL-2 Sturmovik Great Battles , DCS World , ...and in all apps

**Describe the issue**
different mesh fashion image for each eye (debug on)

**Hardware**
Oculus Quest2

**Config and log**
win10x64 20H2 , i5-4670K 4.5Ggz , nvidia GTX1080ti MSI 11Ggb , ssd
![Безымянный](https://user-images.githubusercontent.com/11330258/147372226-fe6c2ce0-7eb0-45e9-93c9-0070b50b4069.jpg)
![Снимок экрана (46)](https://user-images.githubusercontent.com/11330258/147372228-93f21c83-0870-4a09-bbee-6ddb67db709c.png)

1-st screen monitor
2-nd screen HMD
And with debug off - two crescent moons in the left eye remain visible

**Comment by huliqan** [comment]
I disabled  smoothing in graphic settings in 
IL-2 Sturmovik Great Battles
and it's works!!! bugs disappeared!!!
...but doesn't works in DCS :(((

**Comment by fholger** [comment]
Thanks for reporting your results! Now the most important question is, does it actually help improve performance? :)

**Comment by huliqan** [comment]
I'll be write after any testings

**Comment by huliqan** [comment]
Yep! in DCS it works too now!!!)))
I disabled smoothing in driver settings in nvidia Control Panel in 3D settings in Programm settings - DCS
...now testing for performance

**Comment by BrziJoe** [comment]
@huliqan Where is smoothing in nvidia control panel? I do have it in german, but even in youtube videos I can not find it in nvidia settings (i.E. here https://www.youtube.com/watch?v=Q5Xc_UJgoPw )

**Comment by huliqan** [comment]
just turn it off in the game's settings! There only these crescents hang in the menu, they are no longer in flight. But I did not notice the performance gain either

**Comment by huliqan** [comment]
iL-2 msaa off does not help. In menu all ok,but in flight - rings visible
...stopping...new release in testing!!! ver.0.2  no,same too:(
but fsr mode works fine!!!

**Comment by BrziJoe** [comment]
In IL-2 hangar is rendered differently from game. Do not test anything in hangar, just in flight.

**Comment by huliqan** [comment]
Yeah! FSR mode works fine in both (fsr-NIS),but FFR mode in IL-2 works incorrectly:(

**Comment by fholger** [comment]
That's not suprising. FSR and NIS are, for the most part, simple post-processing effects that just work on the final rendered image. Fixed foveated, by contrast, needs to be effective during the entire render of a frame. A lot of things can go wrong as different engines and games do different things during rendering. If either of you has an RTX card, you could try the new VRS mode in the latest version; it has a better chance of working.

**Comment by huliqan** [comment]
In any case, your work is great !!! Thanks!
FFR works in DCS
FSR works in IL-2
